ui/src/app/workflows/components/events-panel.tsx

```diff
@@ -46,6 +47,44 @@ export const EventsPanel = ({namespace, name, kind}: {namespace: string; name: s
         return () => lw.stop();
     }, [showAll, hideNormal]);
 
+    const tableRef = useRef<HTMLDivElement | null>(null);
+
+    useEffect(() => {
+        const calculateTooltips = () => {
+            const table = tableRef.current;
+
+            if (table) {
+                const columns = table.querySelectorAll<HTMLDivElement>('.argo-table-list__row .columns');
+
+                // querySelectorAll does not return an array, it returns a NodeList which is not necessarily
+                // iterable in older browsers.  As of 2/9/21, we target es5 with typescript, meaning it's
+                // not valid to use `for of` loops with NodeList.  When that target is changed or if
+                // downlevelIteration is enabled, we can swap this to use a `for of` loop.
+
+                // tslint:disable-next-line:prefer-for-of
+                for (let i = 0; i < columns.length; i++) {
+                    const col = columns[i];
+
+                    if (col.scrollWidth > col.clientWidth) {
+                        col.title = col.textContent;
+                    } else {
+                        col.title = '';
```

Comment:
Should this be `...` instead? Or something to indicate we're cutting off text?

Reply:
`title` is just the tooltip that will show up when you hover over the element with your mouse.  The browser itself takes care of truncating with `...` when necessary.  Setting this to empty basically just disables the tooltip when the full text is visible in your window